File: backend/codeFiles/custom.js
function greet(name) {
    return "Hello, " + name + "!";
}
const square = (n) => {
    return n * n;
};
function main() {
    let x = 3;
    let y = 8;
    console.log("Square of", x, "is", square(x));
    console.log(greet("John"));
}
main();

File: backend/codeFiles/custom.py
def greet(name):
    return "Hello, " + name + "!"


def square(n):
    return n * n


def main():
    x = 4
    y = 7
    result = square(x)
    message = greet("Alice")
    print("Message:", message)
    print("Square of", x, "is", result)


if __name__ == "__main__":
    main()
